rails-cloud-tasks.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 2.4'
+  spec.required_ruby_version = '>= 2.7'
```

Comment:
This change is really weird 🤔 To fix Rubocop issues you've upgraded the required ruby version.

You have to think that this lib is being used for other APIs.  I don't think we should bump this right now.